homeassistant/components/esphome/climate.py

```diff
@@ -61,6 +61,9 @@
     platform_async_setup_entry,
 )
 
+# climate doesn't support (yet?) the quiet fan mode but esphome does
```

Comment:
https://developers.home-assistant.io/docs/core/entity/climate?_highlight=clim#fan-modes

There are a couple of built-in fan modes, but you're also allowed to use custom fan modes.  We wouldn't expect to add this to the climate platform
```suggestion
```